image/base/redis

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Remove this file.

Move contents to https://github.com/discourse/discourse_docker/blob/master/templates/redis.template.yml#L2-L8